src/containers/InboxPage/InboxPage.js

```diff
@@ -236,11 +241,14 @@ export const InboxPageComponent = props => {
     const transactionProcess = resolveLatestProcessName(process);
     const isBooking = isBookingProcess(transactionProcess);
 
+    const { availabilityType } = foundListingTypeConfig || {};
```

Comment:
There's no need to spread the content of `foundListingTypeConfig` in multiple places. I.e. just do this at the same time as stockType is extracted.